tests/miniconda_install_test.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+
+DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../scripts/newinstall.sh
+. "${DIR}/../scripts/newinstall.sh"
+
+# "stub" out cmd to prevent curl from being run
+# shellcheck disable=SC2034
+cmd="echo"
+
+# test expected usage
+miniconda::install 3 4.2.12 /dne
```

Comment:
`/dne` is just an (ignored) prefix, right? I first looked for it as some sort of argument to the miniconda installer. Worth a comment?